contrib/forms/example2/style.css

```diff
@@ -3,72 +3,72 @@
  */ 
 
 h1 {
-    font-size: 120%;
-    font-weight: bold;
+  font-size: 120%;
+  font-weight: bold;
 }
 
 #form_container {
-    border: 1px solid black;
-    background-color: #ffffcc;
-    width: 750px;
-    padding: 10px;
-    margin: 10px 0px 10px 0px;
+  background-color: #ffc;
+  border: 1px solid black;
+  margin: 10px 0;
```

Comment:
it looks like this is changing the actual margins here (from `10px 0px 10px 0px` to `10px 0`. Is it supposed to do that?

Reply:
@bradymiller This is actually the shorthand version of the exact same line so there's no difference whatsoever.